dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -1328,10 +1328,8 @@ public boolean canCreateNewVersion(Context context, Item item) throws SQLExcepti
     @Override
     public List<MetadataValue> getMetadata(Item item, String schema, String element, String qualifier, String lang,
                                            boolean enableVirtualMetadata) {
-        //Fields of the relation schema are virtual metadata
-        //except for relation.type which is the type of item in the model
-        if (StringUtils.equals(schema, MetadataSchemaEnum.RELATION.getName()) && !StringUtils.equals(element, "type")) {
-
+        // Fields of the relation schema are virtual metadata
```

Comment:
This may not be a correct replacement (although I'm not 100% sure)
However, once https://github.com/DSpace/DSpace/pull/3170 is merged, this part of the code is removed

Reply:
@benbosman ; Good to know.  I'm fine with us working to merge #3170 first, in which case this change can be ignored (as I'll remove it after #3170 gets merged).

FWIW, the original code here is odd anyways (so I'm glad it's being removed).  It seems to be testing a scenario where `relation.type` might exist...that's a non-existing field, as the field is `relationship.type` currently & soon to be `dspace.entity.type`.  In any case, we don't need to dig further here...I'm fine with working to try to merge #3170 first.

Reply:
This has now been resolved by removal of any changes to `ItemServiceImpl` (after rebasing on `main` once #3170 was merged)